storybook/src/docs/components/DataGrid/DataGridExcelExport.stories.tsx

```diff
@@ -0,0 +1,213 @@
+ */
+const TemplateStory: Story = (props) => {
+    const Story = () => {
+        const dataGridProps = useDataGridRemote();
+
+        const variables = {
+            limit: dataGridProps.pageSize,
+            offset: dataGridProps.page * dataGridProps.pageSize,
+            sort: dataGridProps.sortModel[0]?.field,
+            order: dataGridProps.sortModel[0]?.sort,
+        };
+
+        const columns: GridColDef[] = [
+            {
+                field: "mission_name",
+                headerName: "Mission Name",
+            },
+        ];
+
+        const [showMoreMenu, setShowMoreMenu] = useState<boolean>(false);
+        const moreMenuRef = useRef<HTMLButtonElement>(null);
+
+        const query = gql`
+            query LaunchesPast($limit: Int, $offset: Int, $sort: String, $order: String) {
+                launchesPastResult(limit: $limit, offset: $offset, sort: $sort, order: $order) {
+                    data {
+                        id
+                        mission_name
+                    }
+                    result {
+                        totalCount
+                    }
+                }
+            }
+        `;
+
+        const { data, loading, error } = useQuery<GQLQuery, GQLQueryVariables | undefined>(query, {
+            variables,
+        });
+
+        const exportApi = useDataGridExcelExport<ExcelLaunch, GQLExcelQuery, GQLExcelQueryVariables>({
+            columns: [
+                {
+                    field: "mission_name",
+                    headerName: "Mission Name",
+                },
+                {
+                    field: "launch_date_local",
+                    headerName: "Launch Date",
+                    type: "dateTime",
+                },
+            ],
+            variables,
+            query: gql`
+                query LaunchesPastExcel($limit: Int, $offset: Int, $sort: String, $order: String) {
+                    launchesPastResult(limit: $limit, offset: $offset, sort: $sort, order: $order) {
+                        data {
+                            id
+                            mission_name
+                            launch_date_local
+                        }
+                        result {
+                            totalCount
+                        }
+                    }
+                }
+            `,
+            resolveQueryNodes: (data) => data.launchesPastResult.data,
+            totalCount: data?.launchesPastResult.result.totalCount ?? 0,
+            exportOptions: {
+                fileName: "ExampleName",
+            },
+        });
+
+        function DemoToolbar() {
+            return (
+                <Toolbar>
+                    <FillSpace />
+                    <ToolbarActions>
+                        <>
+                            <Button variant="textDark" ref={moreMenuRef} onClick={() => setShowMoreMenu(true)} endIcon={<MoreVertical />}>
```

Comment:
It would be good if we could use CrudMoreActionsMenu here. This would also remove "irrelevant" code from the story.